docs/components/modeler/desktop-modeler/process-applications.md

```diff
@@ -44,25 +50,31 @@ consumer-loan-application/
 └── README.md
 ```
 
-When you open a file in the modeler, the system implicitly determines whether it belongs to a process application by checking for the presence of a `.process-application` file in the same folder or a parent folder. If such a file is found, the process application is implicitly opened, indicated by the blue item in the status bar. All related files within the process application are made accessible for navigation. Similarly, when you close the file, the process application is implicitly closed.
+## Editor support for process applications
+
+When you open a file in Modeler, the system implicitly determines whether it belongs to a process application. It does so by checking for the presence of a `.process-application` file in the same folder or a parent folder. Within a process application, Modeler offers advanced navigation and intelligence.
+
+### Indicating context
+
+Process applications are opened and closed "implicitly": A blue item in the status bar indicates whether a diagram belongs to a process application and makes all related diagrams available for navigation.
 
 <p><img src={OverlayImg} alt="Process application" /></p>
 
 When files of more than one process application are opened they are grouped visually.
 
 <p><img src={GroupingImg} alt="Process application file grouping" /></p>
 
-## Creating a process application
+### Creating a process application
 
-To create a process application, take the following steps:
+Create a process application by creating a `.process-application` file in the root of your project. Alternatively, create it via Modeler UI by taking the following steps:
 
 1. Click **File > New Process Application...**.
 2. Choose a folder.
 3. Click **Select folder**.
 
 A `.process-application` file will be created in the selected folder. Any file within the folder or its subfolders will be treated as part of the process application.
```

Comment:
[❓] Would it make sense to add something around the fact that the folder becomes the project root?
```suggestion
A `.process-application` file will be created in the selected folder which will now serve as the project root. 

Any file within the folder or its subfolders will be treated as part of the process application.
```